### ISSUE TITLE
Placeholder doesn't properly account for borders on table cells, resulting in an ugly "jump" on drag start

### ISSUE THREAD

**Issue by firxworx** [atlassian/react-beautiful-dnd#1887]
I am encountering an issue that I believe is caused by the placeholder provided via the `Droppable` component. I suspect that the placeholder is not properly accounting for borders.

In my case, this results in a 1px "jump" when starting a drag that undermines the otherwise beautiful drag-and-drop experience provided by react-beautiful-dnd.

I have created a working example in a codesandbox where you can see this issue (confirmed on MacOS Chrome v83):
https://codesandbox.io/s/react-material-ui-and-react-beautiful-dnd-uofv4

I am using Material UI tables and have the rows draggable to enable reordering of them. Material UI's default styling has a 1px border-bottom on each `TableCell` (`td`) inside each row. 

The issue is apparent when dragging any table row that isn't the last one. When you start dragging a given row and the placeholder appears, all subsequent rows get bumped down the height of the border. In Material UI, this also causes the last row's border to disappear.

If I add custom styles to remove the border-bottom from all table cells then the "jump" issue does not occur. If I increase the width of the border (e.g to 5px), the content jumps by that amount.

### Expected behavior

There should be no "jump" when a table row that features cells with borders is dragged.

The user experience should be similar to when all borders are removed via CSS.

### Actual behavior

At the start of a drag, the placeholder bumps everything down by the height of the dragged row's borders. 

### Steps to reproduce

Refer to codesandbox: https://codesandbox.io/s/react-material-ui-and-react-beautiful-dnd-uofv4
Reproduced on Chrome v83 on MacOS
Drag to reorder any table row that isn't the last row.

### Versions

Refer to codesandbox, where you can also fork and manipulate versions
React 16.12.0, React Beautiful DnD 13.0.0

### Other comments

Can anyone else please confirm this issue? 

Please advise if there is anything I'm doing wrong or if there are any workarounds.

As an aside, it would be sweet if certain parameters of the placeholder were customizable, or if custom placeholders were more directly supported + documented.

**Comment by firxworx** [comment]
Confirmed that this issue is not just with Material UI tables. 

See the great example from ZenDesk's developers of react-beautiful-dnd to reorder table rows: https://garden.zendesk.com/react-components/tables/#draggable

You can clearly see the same issue. Grab a row from the middle of the table and see the jump on every row below, and observe how the bottom row's bottom border is bumped from view.

Also confirmed the issue also exists in Firefox and Safari on MacOS.

**Comment by coreybruyere** [comment]
@firxworx - Did you ever happen to get this resolved? Having the same issue.

**Comment by firxworx** [comment]
@coreybruyere nope! thanks for confirming its still an issue w/ latest versions

**Comment by goongoguma** [comment]
I have created an example that fixed the issue based on @coreybruyere[firxworx](https://github.com/firxworx)'s code sandbox.
Let me know if there is something should be fixed.
[code sandbox](https://codesandbox.io/s/busy-dhawan-lkq0fh?file=/src/App.js)
